sormas-ui/src/main/java/de/symeda/sormas/ui/immunization/components/fields/pickorcreate/ImmunizationPickOrCreateOption.java

```diff
@@ -0,0 +1,14 @@
+package de.symeda.sormas.ui.immunization.components.fields.pickorcreate;
+
+import com.vaadin.v7.ui.OptionGroup;
+
+import de.symeda.sormas.ui.utils.CssStyles;
+
+public class ImmunizationPickOrCreateOption extends OptionGroup {
```

Comment:
Is this used in multiple places? If not, I don't think we need a separate class for it

Reply:
Yes, it's used in two places.